src/lib/bluetooth/bluetoothbackend.cpp

```diff
@@ -115,8 +115,6 @@ void BluetoothBackend::doStartDiscovery()
 void BluetoothBackend::stopDiscovery()
```

Comment:
Inline commenting so github knows which patch this is about.

As I said before "fix" is very vague verb that hardly ever summarizes the change. Please summarize the change itself and in the description, the benefits and/or rationale.